content/en/docs/Tracing/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Tracing
+description: Tracing Kyverno engine admission requests processing
+weight: 65
+---
+
+## Introduction
+
+Tracing is a method of tracking application requests as they are processed by the application software. When a software is instrumented it produces traces, traces are made of spans hierarchically organised to form a trace. Spans are defined by a start and end time, eventually a parent span, and a number of properties that define the span characteristics (client spans, server spans, etc...).
+
+Tracing is not limited to a single application, as the tracing context can be transmitted on the wire it is possible to collect spans from multiple applications and reconcile them in a single trace.
+
+In the context of Kyverno, requests are usually sent by the Kubernetes API server to the Kyverno service during the admission phase. Kyverno receives and processes admission requests according to the configured policies. Every step in the admission pipeline and during the engine policy processing will produce spans. All clients (Kubernetes client, registry client and cosign client) have also been instrument to produce client spans and transmit the tracing context on the wire.
+
+## Installation and Setup
+
+Tracing requires a backend where Kyverno will send traces. Kyverno uses OpenTelemetry for instrumentation and supports various backends like [Jaeger](https://www.jaegertracing.io/), [Grafana Tempo](https://grafana.com/oss/tempo/) or [Datadog](https://docs.datadoghq.com/tracing/) to name a few.
```

Comment:
I would love to try this with Jaeger. Can you put together a 'quick start' guide of sorts which allow users to see this in action?

Reply:
I will see if I can write a Jaeger walkthrough after the Tempo one is good.

Reply:
Honestly, I wouldn't worry about it. You're welcome to, but the Tempo one is good.

Reply:
I have one almost done.

Reply:
Ok, cool